src/main/java/org/jenkinsci/plugins/durabletask/FileMonitoringTask.java

```diff
@@ -180,6 +181,7 @@ protected FileMonitoringController(FilePath ws) throws IOException, InterruptedE
             CountingOutputStream cos = new CountingOutputStream(transcodedSink);
             try {
                 log.act(new WriteLog(lastLocation, new RemoteOutputStream(cos)));
+                sink.flush();
```

Comment:
What does this have to do with z/OS support?

Reply:
Well, this is due to weird IBM Java behavior on z/OS. Without flushing stream some of the job output isn't timely appears in a log.  No side effects on OpenJDK or Sun Java.